src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/transport/ParsedMessage.kt

```diff
@@ -144,6 +144,8 @@ class ParsedMessage private constructor(
 
 
     companion object {
+        val EMPTY = ParsedMessage(type = "", body = emptyMap())
```

Comment:
Why do you need an empty value for a message?

Reply:
Yes, conn-http expects pair of message: Parsed and Raw. the conn uses empty message when one of them don't come.

Reply:
Then maybe it should use `null` instead of an EMTPY value... The null is a great indicator that we don't have a value

Reply:
conn-http works with messages as they have empty body, metadata and so on